packages/host/app/components/matrix/room.gts

```diff
@@ -93,7 +121,11 @@ export default class Room extends Component<RoomArgs> {
           </div>
         </div>
         {{#each this.messageCardComponents as |Message|}}
-          <Message />
+          {{#if Message.command}}
+            <PatchMessage @command={{Message.command}} />
```

Comment:
isn't "patch" the command in this case? can you tell me what an example `Message.command` would be in this case? otherwise it's awkward to have a hard coded command component, in which case I'm not seeing what the utility of having `PatchMessage` component, since it is literally only wrapping a button and the button's action.

Reply:
This is now extracted